app/src/main/java/com/example/mylittlelibrary/ui/viewModel/BookViewModel.kt

```diff
@@ -5,9 +5,10 @@ import androidx.lifecycle.ViewModel
 import androidx.lifecycle.viewModelScope
 import com.example.mylittlelibrary.data.Book
 import com.example.mylittlelibrary.service.BookService
+import com.example.mylittlelibrary.service.IBookService
 import kotlinx.coroutines.launch
 
-class BookViewModel(var bookService: BookService = BookService()): ViewModel() {
```

Comment:
Added IBookService Interface
https://www.digitalocean.com/community/conceptual_articles/s-o-l-i-d-the-first-five-principles-of-object-oriented-design

Reply:
Good citation.